Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/PyYAML-3.12.egg-info'...
Listing './lib/python3.5/site-packages/adminschUtils'...
Listing './lib/python3.5/site-packages/adminschUtils/AAA'...
Listing './lib/python3.5/site-packages/adminschUtils/config'...
Listing './lib/python3.5/site-packages/adminschUtils/connections'...
Listing './lib/python3.5/site-packages/adminschUtils/sphinx'...
Listing './lib/python3.5/site-packages/adminschUtils/webserver'...
Compiling './lib/python3.5/site-packages/adminschUtils/webserver/webserver.py'...
***   File "./lib/python3.5/site-packages/adminschUtils/webserver/webserver.py", line 27
    __blueprints: Dict[Tuple[str, Optional[str]], Blueprint] = {}
                ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/adminschUtils-0.2.0.egg-info'...
Listing './lib/python3.5/site-packages/aiodns'...
Listing './lib/python3.5/site-packages/aiodns-1.1.1.dist-info'...
Listing './lib/python3.5/site-packages/aiofiles'...
Listing './lib/python3.5/site-packages/aiofiles/threadpool'...
Listing './lib/python3.5/site-packages/aiofiles-0.3.2.dist-info'...
Listing './lib/python3.5/site-packages/aiohttp'...
Listing './lib/python3.5/site-packages/aiohttp-2.3.7.dist-info'...
Listing './lib/python3.5/site-packages/aiozk'...
Listing './lib/python3.5/site-packages/aiozk/protocol'...
Listing './lib/python3.5/site-packages/aiozk/recipes'...
Listing './lib/python3.5/site-packages/aiozk/test'...
Compiling './lib/python3.5/site-packages/aiozk/test/conftest.py'...
***   File "./lib/python3.5/site-packages/aiozk/test/conftest.py", line 36
    print(f'Tree dump: {out}')
                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/aiozk/test/test_client.py'...
***   File "./lib/python3.5/site-packages/aiozk/test/test_client.py", line 14
    child_1 = f'{path}/{uuid.uuid4().hex}'
                                         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/aiozk/test/test_treecache.py'...
***   File "./lib/python3.5/site-packages/aiozk/test/test_treecache.py", line 16
    await self.zk.create(f'{self.path}/{name}')
                                             ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/aiozk/test/test_watchers.py'...
***   File "./lib/python3.5/site-packages/aiozk/test/test_watchers.py", line 79
    yield f'{path}/{uuid.uuid4().hex}'
                                     ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/aiozk-0.5.3.egg-info'...
Listing './lib/python3.5/site-packages/astroid'...
Listing './lib/python3.5/site-packages/astroid/brain'...
Listing './lib/python3.5/site-packages/astroid/interpreter'...
Listing './lib/python3.5/site-packages/astroid/interpreter/_import'...
Listing './lib/python3.5/site-packages/astroid/tests'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/SSL1'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/absimp'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/absimp/sidepackage'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/all.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/all.py", line 7
    def func(): print 'yo'
                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/appl'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/contribute_to_namespace'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/contribute_to_namespace/namespace_pep_420'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/find_test'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/foogle'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/foogle/fax'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/invalid_encoding.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/invalid_encoding.py", line 0
SyntaxError: unknown encoding: lala

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/lmfp'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module.py", line 32
    except ValueError, ex:
                     ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module1abs'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module2.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/module2.py", line 78
    exec 'c = 3'
               ^
SyntaxError: Missing parentheses in call to 'exec'

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/namespace_pep_420'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/noendingnewline.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python2/data/noendingnewline.py", line 21
    print 'a'
            ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/notamodule'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/package/subpackage'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkg_resources_1'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkg_resources_1/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkg_resources_2'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkg_resources_2/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkg_resources_3'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkg_resources_3/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkgutil_1'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkgutil_1/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkgutil_2'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkgutil_2/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkgutil_3'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/path_pkgutil_3/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/unicode_package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python2/data/unicode_package/core'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/SSL1'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/absimp'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/absimp/sidepackage'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/appl'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/contribute_to_namespace'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/contribute_to_namespace/namespace_pep_420'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/find_test'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/foogle'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/foogle/fax'...
Compiling './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/invalid_encoding.py'...
***   File "./lib/python3.5/site-packages/astroid/tests/testdata/python3/data/invalid_encoding.py", line 0
SyntaxError: unknown encoding: lala

Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/lmfp'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/module1abs'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/namespace_pep_420'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/notamodule'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/package/subpackage'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkg_resources_1'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkg_resources_1/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkg_resources_2'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkg_resources_2/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkg_resources_3'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkg_resources_3/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkgutil_1'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkgutil_1/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkgutil_2'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkgutil_2/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkgutil_3'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/path_pkgutil_3/package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/unicode_package'...
Listing './lib/python3.5/site-packages/astroid/tests/testdata/python3/data/unicode_package/core'...
Listing './lib/python3.5/site-packages/astroid-1.6.0.dist-info'...
Listing './lib/python3.5/site-packages/async_timeout'...
Listing './lib/python3.5/site-packages/async_timeout-2.0.0.dist-info'...
Listing './lib/python3.5/site-packages/cchardet'...
Listing './lib/python3.5/site-packages/cchardet-2.1.1.dist-info'...
Listing './lib/python3.5/site-packages/chardet'...
Listing './lib/python3.5/site-packages/chardet/cli'...
Listing './lib/python3.5/site-packages/chardet-3.0.4.dist-info'...
Listing './lib/python3.5/site-packages/frozendict'...
Listing './lib/python3.5/site-packages/frozendict-1.2.egg-info'...
Listing './lib/python3.5/site-packages/httptools'...
Listing './lib/python3.5/site-packages/httptools/parser'...
Listing './lib/python3.5/site-packages/httptools-0.0.10.egg-info'...
Listing './lib/python3.5/site-packages/idna'...
Listing './lib/python3.5/site-packages/idna-2.6.dist-info'...
Listing './lib/python3.5/site-packages/isort'...
Listing './lib/python3.5/site-packages/isort-4.2.15.dist-info'...
Listing './lib/python3.5/site-packages/lazy_object_proxy'...
Listing './lib/python3.5/site-packages/lazy_object_proxy-1.3.1.dist-info'...
Listing './lib/python3.5/site-packages/mccabe-0.6.1.dist-info'...
Listing './lib/python3.5/site-packages/multidict'...
Listing './lib/python3.5/site-packages/multidict-3.3.2.dist-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/pycares'...
Listing './lib/python3.5/site-packages/pycares-2.3.0.dist-info'...
Listing './lib/python3.5/site-packages/pylint'...
Listing './lib/python3.5/site-packages/pylint/checkers'...
Listing './lib/python3.5/site-packages/pylint/extensions'...
Listing './lib/python3.5/site-packages/pylint/pyreverse'...
Listing './lib/python3.5/site-packages/pylint/reporters'...
Listing './lib/python3.5/site-packages/pylint/reporters/ureports'...
Listing './lib/python3.5/site-packages/pylint/test'...
Listing './lib/python3.5/site-packages/pylint/test/acceptance'...
Listing './lib/python3.5/site-packages/pylint/test/data'...
Listing './lib/python3.5/site-packages/pylint/test/extensions'...
Listing './lib/python3.5/site-packages/pylint/test/extensions/data'...
Listing './lib/python3.5/site-packages/pylint/test/functional'...
Compiling './lib/python3.5/site-packages/pylint/test/functional/anomalous_unicode_escape_py2.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/anomalous_unicode_escape_py2.py", line 17
    RAW_UNICODE = ur"\u0062\n"
                             ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/functional/async_functions.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/async_functions.py", line 57
    async def func(a, a, b=[]):
         ^
SyntaxError: duplicate argument 'a' in function definition

Compiling './lib/python3.5/site-packages/pylint/test/functional/bad_except_order.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/bad_except_order.py", line 27
    __revision__ += 1
                   ^
SyntaxError: default 'except:' must be last

Compiling './lib/python3.5/site-packages/pylint/test/functional/class_members_py27.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/class_members_py27.py", line 24
    print self.nonexistent
             ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/pylint/test/functional/continue_in_finally.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/continue_in_finally.py", line 9
    continue # [continue-in-finally]
    ^
SyntaxError: 'continue' not supported inside 'finally' clause

Compiling './lib/python3.5/site-packages/pylint/test/functional/disable_msg_github_issue_1389.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/disable_msg_github_issue_1389.py", line 10
    place: PlaceId
         ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/functional/duplicate_argument_name.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/duplicate_argument_name.py", line 4
    def foo1(_, _): # [duplicate-argument-name]
                                              ^
SyntaxError: duplicate argument '_' in function definition

Compiling './lib/python3.5/site-packages/pylint/test/functional/exec_used_py2.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/exec_used_py2.py", line 3
    exec 'a = __revision__' # [exec-used]
                          ^
SyntaxError: Missing parentheses in call to 'exec'

Compiling './lib/python3.5/site-packages/pylint/test/functional/formatted_string_literal_with_if_py36.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/formatted_string_literal_with_if_py36.py", line 4
    f'{"+" if True else "-"}'
                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/functional/invalid_star_assignment_target.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/invalid_star_assignment_target.py", line 3
    *FIRST = [1, 2, 3] # [invalid-star-assignment-target]
                                                        ^
SyntaxError: starred assignment target must be in a list or tuple

Compiling './lib/python3.5/site-packages/pylint/test/functional/iterable_context_py2.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/iterable_context_py2.py", line 16
    print i
          ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/pylint/test/functional/misplaced_future.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/misplaced_future.py", line 3
    from __future__ import print_function # [misplaced-future]
                                                             ^
SyntaxError: from __future__ imports must occur at the beginning of the file

Compiling './lib/python3.5/site-packages/pylint/test/functional/nonlocal_and_global.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/nonlocal_and_global.py", line 5
    nonlocal missing
    ^
SyntaxError: name 'missing' is nonlocal and global

Compiling './lib/python3.5/site-packages/pylint/test/functional/nonlocal_without_binding.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/nonlocal_without_binding.py", line 22
    nonlocal b  # [nonlocal-without-binding]
    ^
SyntaxError: no binding for nonlocal 'b' found

Compiling './lib/python3.5/site-packages/pylint/test/functional/not_in_loop.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/not_in_loop.py", line 7
    continue # [not-in-loop]
    ^
SyntaxError: 'continue' not properly in loop

Compiling './lib/python3.5/site-packages/pylint/test/functional/old_division_manually.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/old_division_manually.py", line 2
    print 1 / 3
          ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/pylint/test/functional/print_always_warns.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/print_always_warns.py", line 8
    print "Python 3 fails to parse print statements." # [print-statement]
                                                    ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/pylint/test/functional/redefine_in_handler.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/redefine_in_handler.py", line 19
    except KeyError, exceptions.RuntimeError: # [redefine-in-handler]
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/functional/return_outside_function.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/return_outside_function.py", line 2
    return # [return-outside-function]
                                     ^
SyntaxError: 'return' outside function

Compiling './lib/python3.5/site-packages/pylint/test/functional/star_needs_assignment_target.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/star_needs_assignment_target.py", line 3
    FIRST = *[1, 2]  # [star-needs-assignment-target]
           ^
SyntaxError: can't use starred expression here

Compiling './lib/python3.5/site-packages/pylint/test/functional/star_needs_assignment_target_py35.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/star_needs_assignment_target_py35.py", line 15
    UNPACK_IN_COMP = {elem for elem in (*range(10))} # [star-needs-assignment-target]
                                       ^
SyntaxError: can't use starred expression here

Compiling './lib/python3.5/site-packages/pylint/test/functional/syntax_error.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/syntax_error.py", line 1
    def toto # [syntax-error]
                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/functional/syntax_error_jython.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/syntax_error_jython.py", line 1
    def toto # [syntax-error]
                            ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/functional/too_many_star_expressions.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/too_many_star_expressions.py", line 3
    *FIRST, *SECOND = [1, 2, 3] # [too-many-star-expressions]
                                                            ^
SyntaxError: two starred expressions in assignment

Compiling './lib/python3.5/site-packages/pylint/test/functional/unpacked_exceptions.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/unpacked_exceptions.py", line 7
    except IOError, (errno, message):  # [unpacking-in-except]
                  ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/functional/yield_from_outside_func.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/yield_from_outside_func.py", line 2
    yield from [1, 2]  # [yield-outside-function]
                                                ^
SyntaxError: 'yield' outside function

Compiling './lib/python3.5/site-packages/pylint/test/functional/yield_inside_async_function.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/yield_inside_async_function.py", line 11
    yield 42 # [yield-inside-async-function]
    ^
SyntaxError: 'yield' inside async function

Compiling './lib/python3.5/site-packages/pylint/test/functional/yield_inside_async_function_py36.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/yield_inside_async_function_py36.py", line 11
    yield 42
    ^
SyntaxError: 'yield' inside async function

Compiling './lib/python3.5/site-packages/pylint/test/functional/yield_outside_func.py'...
***   File "./lib/python3.5/site-packages/pylint/test/functional/yield_outside_func.py", line 2
    yield 1  # [yield-outside-function]
                                      ^
SyntaxError: 'yield' outside function

Listing './lib/python3.5/site-packages/pylint/test/input'...
Compiling './lib/python3.5/site-packages/pylint/test/input/func_w0122_py_30.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_w0122_py_30.py", line 5
    exec 'a = __revision__'
                          ^
SyntaxError: Missing parentheses in call to 'exec'

Compiling './lib/python3.5/site-packages/pylint/test/input/func_w0332_py_30.py'...
***   File "./lib/python3.5/site-packages/pylint/test/input/func_w0332_py_30.py", line 4
    __revision__ = 1l
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pylint/test/input/func_w0401_package'...
Listing './lib/python3.5/site-packages/pylint/test/messages'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/absimp'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/bad_package'...
Compiling './lib/python3.5/site-packages/pylint/test/regrtest_data/bad_package/__init__.py'...
***   File "./lib/python3.5/site-packages/pylint/test/regrtest_data/bad_package/__init__.py", line 2
    raise missing.Missing..
                          ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/beyond_top'...
Compiling './lib/python3.5/site-packages/pylint/test/regrtest_data/descriptor_crash.py'...
***   File "./lib/python3.5/site-packages/pylint/test/regrtest_data/descriptor_crash.py", line 20
    print p.getPage(sys.argv[1])
          ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/dummy'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/dummy_plugin'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/package'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/package/subpackage'...
Listing './lib/python3.5/site-packages/pylint/test/regrtest_data/package_all'...
Compiling './lib/python3.5/site-packages/pylint/test/regrtest_data/py3k_error_flag.py'...
***   File "./lib/python3.5/site-packages/pylint/test/regrtest_data/py3k_error_flag.py", line 4
    raise Exception, 1  # Error emitted here with the Python 3 checker.
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/regrtest_data/py3k_errors_and_warnings.py'...
***   File "./lib/python3.5/site-packages/pylint/test/regrtest_data/py3k_errors_and_warnings.py", line 8
    raise Exception, 1
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/pylint/test/regrtest_data/syntax_error.py'...
***   File "./lib/python3.5/site-packages/pylint/test/regrtest_data/syntax_error.py", line 1
    class A extends B {}
                  ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/pylint-1.8.1.dist-info'...
Listing './lib/python3.5/site-packages/python_json_logger-0.1.8.dist-info'...
Listing './lib/python3.5/site-packages/pythonjsonlogger'...
Listing './lib/python3.5/site-packages/recursive_dictionary_update'...
Listing './lib/python3.5/site-packages/recursive_dictionary_update-0.1.0.dist-info'...
Listing './lib/python3.5/site-packages/sanic'...
Listing './lib/python3.5/site-packages/sanic-0.7.0.dist-info'...
Listing './lib/python3.5/site-packages/sanic_openapi'...
Listing './lib/python3.5/site-packages/sanic_openapi/ui'...
Listing './lib/python3.5/site-packages/sanic_openapi-0.4.0.egg-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/six-1.11.0.dist-info'...
Listing './lib/python3.5/site-packages/tipsi_tools'...
Listing './lib/python3.5/site-packages/tipsi_tools/django'...
Listing './lib/python3.5/site-packages/tipsi_tools/doc_utils'...
Listing './lib/python3.5/site-packages/tipsi_tools/doc_utils/tipsi_sphinx'...
Listing './lib/python3.5/site-packages/tipsi_tools/drf'...
Listing './lib/python3.5/site-packages/tipsi_tools/scripts'...
Listing './lib/python3.5/site-packages/tipsi_tools/testing'...
Listing './lib/python3.5/site-packages/tipsi_tools/testing/drf'...
Listing './lib/python3.5/site-packages/tipsi_tools-1.17.0.egg-info'...
Listing './lib/python3.5/site-packages/ujson-1.35.egg-info'...
Listing './lib/python3.5/site-packages/uvloop'...
Listing './lib/python3.5/site-packages/uvloop/handles'...
Listing './lib/python3.5/site-packages/uvloop/includes'...
Listing './lib/python3.5/site-packages/uvloop-0.9.1.dist-info'...
Listing './lib/python3.5/site-packages/voluptuous'...
Listing './lib/python3.5/site-packages/voluptuous-0.10.5.egg-info'...
Listing './lib/python3.5/site-packages/websockets'...
Listing './lib/python3.5/site-packages/websockets/extensions'...
Listing './lib/python3.5/site-packages/websockets/py35'...
Listing './lib/python3.5/site-packages/websockets-4.0.1.dist-info'...
Listing './lib/python3.5/site-packages/wrapt'...
Listing './lib/python3.5/site-packages/wrapt-1.10.11.egg-info'...
Listing './lib/python3.5/site-packages/yaml'...
Listing './lib/python3.5/site-packages/yarl'...
Listing './lib/python3.5/site-packages/yarl-0.18.0.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
